pallets/nfts/src/weights.rs

```diff
@@ -732,19 +734,21 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	/// Storage: `Nfts::ItemConfigOf` (r:1 w:0)
 	/// Proof: `Nfts::ItemConfigOf` (`max_values`: None, `max_size`: Some(48), added: 2523, mode: `MaxEncodedLen`)
 	/// Storage: `Nfts::AccountBalance` (r:2 w:2)
-	/// Proof: `Nfts::AccountBalance` (`max_values`: None, `max_size`: Some(72), added: 2547, mode: `MaxEncodedLen`)
+	/// Proof: `Nfts::AccountBalance` (`max_values`: None, `max_size`: Some(120), added: 2595, mode: `MaxEncodedLen`)
+	/// Storage: `System::Account` (r:1 w:1)
```

Comment:
As above.

Reply:
The `buy_item` implementation calls `do_transfer` to transfer from the buyer to the dest account. Hence, Takes reads and writes similar to the `transfer()` dispatchable. 

- `r: 1`: Read the balance of the `dest` account with `T::Currency::can_reserve` to identify the depositor of the call. 
- `w: 1`: Reserve from the `depositor` account. Either the `caller` or the `dest` account.